src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import StackedBarChart from 'app/components/stackedBarChart';
+import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
+import DateTime from 'app/components/dateTime';
+import LoadingError from 'app/components/loadingError';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {intcomma} from 'app/utils';
+import {t} from 'app/locale';
+import {SentryApp} from 'app/types';
+
+type Props = AsyncView['props'];
+
+type State = AsyncView['state'] & {
+  stats: any;
+  errors: any;
+  app: SentryApp | null;
+};
+
+export default class SentryApplicationDashboard extends AsyncView<Props, State> {
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      stats: {},
+      errors: [],
+      app: null,
+    };
+  }
+
+  getEndpoints(): Array<[string, string, any] | [string, string]> {
+    const {appSlug} = this.props.params;
+    if (appSlug) {
+      // Default time range for now: 90 days ago to now
+      const now = Math.floor(new Date().getTime() / 1000);
+      const ninety_days_ago = 3600 * 24 * 90;
+      return [
+        [
+          'stats',
+          `/sentry-apps/${appSlug}/stats/`,
+          {query: {since: now - ninety_days_ago, until: now}},
+        ],
+        ['errors', `/sentry-apps/${appSlug}/errors/`],
+        ['app', `/sentry-apps/${appSlug}/`],
+      ];
+    }
+    return [];
+  }
+
+  getTitle() {
+    return t('Integration Dashboard');
+  }
+
+  renderInstallData() {
+    const {total_uninstalls, total_installs} = this.state.stats;
+    return (
+      <React.Fragment>
+        <h5>{t('Installation Data')}</h5>
+        <Row>
+          <StatsSection>
+            <StatsHeader>{t('Total installs')}</StatsHeader>
+            <p>{total_installs}</p>
+          </StatsSection>
+          <StatsSection>
+            <StatsHeader>{t('Total uninstalls')}</StatsHeader>
+            <p>{total_uninstalls}</p>
+          </StatsSection>
+        </Row>
+        {this.renderInstallCharts()}
+      </React.Fragment>
+    );
+  }
+
+  renderTooltip(point, _pointIdx, chart) {
+    const timeLabel = chart.getTimeLabel(point);
+    const [installed, uninstalled] = point.y;
+
+    return (
+      <div style={{width: '150px'}}>
+        <div className="time-label">{timeLabel}</div>
+        <div className="value-label">
+          {intcomma(installed)} {`install${installed !== 1 ? 's' : ''}`}
```

Comment:
@nolachen Looks like we have translations for the rest of the page but not here. I think that's OK since all of our app developers should be proficient in English but I just want to point it out.